toc.json

```diff
@@ -141,7 +141,7 @@
     "key": "simcore/services/dynamic/mapcore-widget",
     "name": "Mapcore Flatmap",
     "type": "dynamic",
-    "version": "1.0.0"
+    "version": "0.1.22"
```

Comment:
that is probably wrong

Reply:
It's correct, no problem. Flatmap widget was never released with 1.0.0 version.